TAO/NEWS

```diff
@@ -1,3 +1,16 @@
+USER VISIBLE CHANGES BETWEEN TAO-2.5.12 and TAO-2.5.13
+======================================================
+
+. TAO IDL:
+
+  . Version of IDL specification being used is available as the
+    `__TAO_IDL_IDL_VERSION` preprocessor macro.
+
+  . Added `--preprocessor-input` option to `tao_idl` to make behavior of
+    `#include ".."` consistent with C and C++. This is not enabled by default
+    like it is in TAO 3, though. For details, see the `--preprocessor-input`
+    argument in `docs/compiler.html`.
```

Comment:
I don't know if the TAO 3 note is needed here.  The scope of this section is TAO 2.

Reply:
Maybe, but it is a behavior difference between the two, so I feel like it has to be mentioned somewhere.

Reply:
Does it require time travel to have TAO 2 know what TAO 3 does?